hudi-spark/src/test/scala/org/apache/hudi/TestDataSourceDefaults.scala

```diff
@@ -34,13 +36,28 @@ import org.scalatest.Assertions.fail
 class TestDataSourceDefaults {
 
   val schema = SchemaTestUtil.getComplexEvolvedSchema
+  val structType = AvroConversionUtils.convertAvroSchemaToStructType(schema)
```

Comment:
Note to reviewer: this is the test class where all key generators are tested for Row apis as well. Found new test classes for each key gen after rebasing. Yet to add tests to these new key gen test classes for Row based apis.

Reply:
@nsivabalan can you please file a JIRA for this?

Reply:
https://issues.apache.org/jira/browse/HUDI-1179